desktop/src/app/components/app.component.ts

```diff
@@ -131,4 +141,24 @@ export class AppComponent {
         document.addEventListener('dragover', event => event.preventDefault());
         document.addEventListener('drop', event => event.preventDefault());
     }
+
+    public async promptEditorName() {
+        this.menus.setContext(MenuContext.MODAL);
+
+        try {
+            const modalRef: NgbModalRef = this.modalService.open(
+                UpdateEditorComponent, { animation: false }
+            );
+            
+            //await modalRef.result;
+            return true;
+        } catch (_) {
+            return false;
+        } finally {
+            this.menus.setContext(MenuContext.DEFAULT);
+        }
+        
+        
+        //router.navigate(["settings"])
+    }
```

Comment:
@tkleinke würdest du das hier lassen oder lieber irgendwo anders einordnen?  Die Funktion wird einmal direkt von der AppComponent getriggered (beim Start, wenn username==anonymous, aber auch wenn man später oben in der Navbar auf das User Icon bzw. die Initialen klickt.

Reply:
Da es ja auch aus der TaskbarEditorComponent heraus aufgerufen wird, sollte es in einen Service, der von beiden Komponenten aufgerufen wird. Ich würde sagen, die Funktion kann in den ProjectModalLauncher, daraus wird auch das Synchronisations-Modal aufgerufen. Evtl. sollte da aber auch eher in die Settings weitergeleitet werden, ansonsten gibt es zwei verschiedene Stellen, an der die gleiche Einstellung vorgenommen werden kann.